Monika After Story/game/pong.rpy

```diff
@@ -27,6 +27,15 @@
                 self.BALL_HEIGHT = 15
                 self.COURT_TOP = 124
                 self.COURT_BOTTOM = 654
+                
+                # Other variables
+                self.COURT_WIDTH = 1280
+                self.COURT_HEIGHT = 720
+                self.BALL_MAX_SPEED = 4000.0
+                self.PADDLE_X_PLAYER = self.COURT_WIDTH * 0.1
```

Comment:
rather than do this `* 0.1` and `* 0.9` thing, can you just hardcode the numbers here, then use them for the render positions below?

Reply:
Sure, it is not like the game would have ever gotten different resolutions.
I'm going to put the real numbers in then.

Reply:
If you have any feedback on the gameplay please let me know as well.